### PR TITLE
WR-12 #12

### LINKED ISSUE #12: Splash screen

### PR DESCRIPTION
**Pull request by JakubiM**
#12

### CODE REVIEW

File: app/src/main/java/com/example/workoutrival/SplashScreen.kt [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+class SplashScreen : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        window.requestFeature(Window.FEATURE_NO_TITLE)
+        window.setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN, WindowManager.LayoutParams.FLAG_FULLSCREEN)
+        setContentView(R.layout.activity_splash_screen)
+
+        Handler().postDelayed({
+            startActivity(Intent(this@SplashScreen, MainActivity::class.java))
```
Comment (PureFour) [inline app/src/main/java/com/example/workoutrival/SplashScreen.kt:19]: Please check service availability...
You may use this endpoint: **http://localhost/actuator/health**  which returns for example: 
```
{
...
     "status" : "UP"
...
}
```

File: app/src/main/res/layout/activity_splash_screen.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:tools="http://schemas.android.com/tools"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        tools:context=".SplashScreen">
+    <ImageView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content" app:srcCompat="@drawable/splash_screen"
+            android:id="@+id/imageView2"
```
Comment (raddyr17) [inline app/src/main/res/layout/activity_splash_screen.xml:12]: Niepotrzebne id.

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -9,13 +9,17 @@
...
+        <activity android:name=".SplashScreen"
+                  android:theme="@style/Theme.AppCompat.Light.NoActionBar">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN"/>
 
                 <category android:name="android.intent.category.LAUNCHER"/>
             </intent-filter>
         </activity>
+        <activity android:name=".MainActivity">
+
```
Comment (raddyr17) [inline app/src/main/AndroidManifest.xml:21]: powinno być  ... ".MainActivity"/>

Comment (raddyr17) [reply app/src/main/AndroidManifest.xml]: <activity android:name=".MainActivity">

**Review by raddyr17 — CHANGES_REQUESTED** [review]
Dwie poprawki potem to merge

**Pull request closed without merging**